content/code_of_conduct.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Code Of Conduct
+image: images/tardis_banner.jpg
+image_alt: TARDIS Banner
+seo:
+  title: Code Of Conduct
+  description: Code Of Conduct of the TARDIS Organisation
```

Comment:
Add something meaningful (maybe 1st line summary of Code of Conduct) here

And everywhere below for description

Reply:
I don't know what you want with this? Should I just modify the description to be a summary?

Reply:
Yes modify all `description` values - make it short 1-2 sentences summary of what Code of Conduct is